src/main/io/osd.c

```diff
@@ -202,6 +202,13 @@ static const uint8_t osdElementDisplayOrder[] = {
 
 PG_REGISTER_WITH_RESET_FN(osdConfig_t, osdConfig, PG_OSD_CONFIG, 3);
 
+static void osdDisplayCenteredMessage(int row, const char *message)
```

Comment:
I don't think making this into a permanent facility. We should rather convert the existing 'ARMED' text into a facility that can be positioned like all the other elements, and default it to the existing position.

Reply:
I didn't really change anything in regards to the "ARMED" message other than make it more nicely centered.  Before it was simply hardcoded into position:
```
    displayWrite(osdDisplayPort, 12, 7, "ARMED");
```
So if we want to make the "ARMED" message configurable that can still be done - but should probably be it's own PR and will also require configurator changes.  The bad thing with making it configurable is that in the current scheme the user could choose to not display it.  Not sure that would be a good change.  Currently it clears the display and displays the message for 0.5s on every arming.